### ISSUE TITLE
Writing assignments

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy-paper#30]
Here are some writing assignments. I'm basing this on what people have [volunteered to write](https://github.com/sympy/sympy-paper/wiki/Who-Will-Write-What). 

Me: The architecture section.

@fredrik-johansson: The numerics section.

@certik: The limits/Gruntz algorithm section.

@aktech and @hargup: The solvers section.

@Sumith1896: The logic section.

@moorepants: The physics section. 

@ashutoshsaboo: SymPy Gamma and Live (already started)

@scopatz: features table. 

@Upabjojr: basic operations (in features)

@shivamvats and @leosartaj: series

Not represented from the current outline
- Calculus
- Matrices
- Risch algorithm
- Polynomials

Some people have already volunteered to write some of these sections but they already have a writing assignment above, and I don't want to put too much work on anyone. If you write your assignment and some other section, go ahead (but please "claim" it first so that we don't duplicate work). 

**Have your initial text in by Wednesday, April 13.** Note that this is just a first draft, so don't worry about refinements or organization too much. It will probably will be easiest if each person writes their part as a separate file. 

Beyond this there will be further deadlines for reviewing.

I apologize if this is a rushed schedule. The goal is to have a draft to submit by May 1.

**Comment by moorepants** [comment]
Thanks. It's good to have deadlines :)

**Comment by amitsaha** [comment]
Hi Aaron, i think you have the wrong Amit. I am not knowledgeable enough about the details to be a major contributor. I will however try to go through what others write and see if i can give any meaningful feedback.

**Comment by asmeurer** [comment]
Ah, super apology for that. I meant @aktech, who volunteered to write that section.

**Comment by ashutoshsaboo** [comment]
@asmeurer Yeah! This seems good! I'll mostly write about Risch Algorithm or Group Theory implementation as well once I finish with SymPy Gamma and SymPy Live. 

BTW are we including the group theory implementation in this paper?

**Comment by asmeurer** [comment]
I don't think we should go into much detail on it, since group theory is not a big focus of SymPy.

**Comment by ashutoshsaboo** [comment]
Ohh I understand @asmeurer ! :+1: I'll complete the **Other Projects that use SymPy** section, by listing details about SymPy Gamma and Live (which I've already started), and mentioning other CASs and other apps/libraries that also use SymPy (as discussed here - #28 ). So is that fine? @asmeurer

**Comment by Upabjojr** [comment]
Pattern matching has been completely ignored?

**Comment by asmeurer** [comment]
I personally don't see pattern matching as one of the key features of SymPy. SymPy isn't designed around it like Mathematica, and the pattern matching that exists in SymPy is relatively weak. However, if you want to write something up on it, feel free.

**Comment by ashutoshsaboo** [comment]
@asmeurer I also have already written about "Other Projects that use SymPy" - fixes #28  (apart from Live and Gamma) here - #31 .

Also, are there any changes required in #31 and #29 ?
